llvm/test/CodeGen/DirectX/splitdouble.ll

```diff
@@ -0,0 +1,77 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
Are these assertions actually autogenerated? They look like they've been written by hand to me (which is fine, but if that's the case, please remove the misleading comment)